src/App.vue

```diff
@@ -69,8 +73,10 @@ export default defineComponent({
     const { toggle: toggleTheme, clockColors } = useTheme()
     const { storage: preferences } = useLocalStorage('pref', { ampm: false })
     const { storage: timeZones } = useLocalStorage<TimeZone[]>('timezones', [])
-    const { now: nowDate } = useNowDate()
     
+    
+    const sliderValue = ref(12);
```

Comment:
I think this value must be in pinia store. We can use it in TimeZoneCalculationCard.